<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dtEnterKey</class>
 <widget class="QDialog" name="dtEnterKey">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter key</string>
  </property>
  <widget class="QLabel" name="mainLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Double Transposition Key:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="rowKeyLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>6</number>
   </property>
   <property name="text">
    <string>Enter row keys (separated by a coma/','):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rowKeyLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="colKeyLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter column keys (separated by a coma/','):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="colKeyLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
